ssl/t1_lib.c

```diff
@@ -3170,10 +3170,11 @@ static int tls_decrypt_ticket(SSL *s, const unsigned char *etick,
     p = sdec;
 
     sess = d2i_SSL_SESSION(NULL, &p, slen);
+    slen -= p - sdec;
```

Comment:
It turns out that VMS has no issues here... to my surprise, but I'm not complaining!